tardis/io/model_reader.py

```diff
@@ -235,6 +244,20 @@ def read_artis_density(fname):
     return time_of_model, velocity, mean_density
 
 
+def read_cmfgen_density(fname):
+
+    df = pd.read_csv(fname, comment='#', delimiter='\s+', skiprows=1)
+    velocity = (df['Velocity'].values[::-1] * u.km / u.s).to('cm/s')
```

Comment:
@vg3095 if you reverse the velocity - you have to reverse all the other columns as well. They all belong together.